CHANGES.rst

```diff
@@ -172,6 +172,12 @@ Bug Fixes
 
 - ``astropy.utils``
 
+  - Fix to allow the C-based _fast_iterparse() VOTable XML parser to
+    relloc() its buffers instead of overflowing them.  [#5824] [#5869]
```

Comment:
@bsipocz, @pllim: could I double check the preference/suggestion.  Is it one of:

1. `… [#5869]`
2. `… #5824 [#5869]`
3. _something else_